submissions/onyemesimChidiebereSabinus/html/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My e-Commerce site</title>
+</head>
+<body>
+
+  <H1> LOUIS     U.S     USED    CAR     SALES</H1>
+  <H4>...<em>authorised to explore</em></H4>
+    ...
+    <h1><em>We hold you by the hand and lead you to purchase and manage
+         your american used car at the best price you 
+    ,count on us, Do the ordering and leave the rest to us</em>
+    </h1>
+                 <img src="https://ford-h.assetsadobe.com/is/image/content/dam/vdm_ford/live/en_us/ford/nameplate/mustang/2020/collections/dm/20_FRD_MST_40830.tif?croppathe=1_3x2&wid=1440">
```

Comment:
I think the images should be downloaded especially if your are working offline.
You can create a folder, `images` in your html folder  and add all images that will be used in your project there.